bears/yaml/TravisLintBear.py

```diff
@@ -24,6 +26,29 @@ class TravisLintBear:
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Formatting', 'Syntax'}
     SEE_MORE = 'https://docs.travis-ci.com/user/travis-lint'
+    CHECK_CONNECTION_URL = 'https://travis-ci.org/'
+
+    @classmethod
+    def check_prerequisites(cls):  # pragma: no cover
+        url_status = cls.get_status_of_url(cls.CHECK_CONNECTION_URL)
+        base_check = super().check_prerequisites()
+        if base_check is not True:
+            return base_check
+        try:
+            if url_status is None:
+                return 'You are not connected to the internet.'
+            else:
+                url_status.raise_for_status()
```

Comment:
From the docs, 
> If we made a bad request (a 4XX client error or 5XX server error response), we can raise it with Response.raise_for_status():

More info here -> http://docs.python-requests.org/en/master/user/quickstart/#response-status-codes